test/include.test.js

```diff
@@ -190,19 +190,322 @@ describe('include', function () {
           relation: 'posts', scope: {
             fields: ['title'], include: ['author'],
             order: 'title DESC',
-            limit: 2
+            limit: 1
           }
         }
       },
-      limit: 1
+      limit: 2
     }, function(err, passports) {
       if (err) return done(err);
-      passports.length.should.equal(1);
-      passports[0].toJSON().owner.posts.length.should.equal(2);
+
+      passports.length.should.equal(2);
+      var posts1 = passports[0].toJSON().owner.posts;
+      posts1.length.should.equal(1);
+      posts1[0].title.should.equal('Post C');
+      var posts2 = passports[1].toJSON().owner.posts;
+      posts2.length.should.equal(1);
+      posts2[0].title.should.equal('Post D');
+
       done();
     });
   });
 
+  describe('inq limit', function() {
+    before(function() {
+      Passport.dataSource.settings.inqLimit = 2;
+    });
+
+    after(function() {
+      delete Passport.dataSource.settings.inqLimit;
+    });
+
+    it('should support include by pagination', function(done) {
+      Passport.find({
+        include: {
+          owner: {
+            relation: 'posts',
+            scope: {
+              fields: ['title'], include: ['author'],
+              order: 'title ASC'
```

Comment:
That test was already there before I took over the PR, but the `pagination` it's refering to is inside the impl and set by `inqLimit = 2` in the before block. I added a comment above to make it a bit more clear.